package.json

```diff
@@ -55,6 +55,7 @@
     "json-stable-stringify": "^1.0.0",
     "levn": "^0.3.0",
     "lodash": "^4.0.0",
+    "minimatch": "^3.0.0",
```

Comment:
We already have a dependency on `glob`, which we use for parsing CLI options. Can you use that instead of adding another glob-parsing dependency?